DesktopModules/Vanjaro/Core/Library/Factories/CacheFactory.cs

```diff
@@ -26,7 +26,8 @@ public class Keys
                 public const string ThemeManager = Prefix + "ThemeManager";
                 public const string Theme = Prefix + "Theme";
                 internal const string OAuthClients = Prefix + "OAuthClients";
-                internal const string IOAuthClient_Extension = Prefix + "IOAuthClient_Extension";                
+                internal const string IOAuthClient_Extension = Prefix + "IOAuthClient_Extension";
+                internal const string CustomPermission = Prefix + "CustomPermission";
```

Comment:
Same. Let's call this either EntityPermission or SectionPermission based on reusability.